Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -1092,6 +1113,11 @@ def get_report_summary_command(
 
     for single_id in argToList(ids):
         response = client.get_report_summary(single_id)
+        if response.get('resources'):
+            # We can extract the error from the response object under resources
```

Comment:
same here, we want to explain to the next dev why we are looking inside resources when we have error section.